src/desktop/components/main_layout/templates/scripts.jade

```diff
@@ -10,6 +10,7 @@ script( src=asset("/assets/legacy-common-jquery.js") )
 script( src=asset("/assets/legacy-common-react.js") )
 script( src=asset("/assets/legacy-common-utility.js") )
 script( src=asset("/assets/legacy-artsy.js") )
+script( id='ze-snippet' src='https://static.zdassets.com/ekr/snippet.js?key=#{sd.ZENDESK_KEY}' )
```

Comment:
mounting the Zendesk script tag here so needn't do it on every single page we want to render the chat button

Reply:
So does this mean that the zendesk code will be embedded on pages that don't use the button/

Reply:
Maybe we may want to conditionally render this based on whether it's an auction url? Or even at the script tag in the body of auctions page versus the main layout for all pages?

Reply:
> Or even at the script tag in the body of auctions page

This is what I was thinking! Another way of going about it would be to add the script tag directly to the ZenDesk component so when a user adds the component it automatically comes with the embed script.

Reply:
I was looking to find where the zendesk script is loaded in the order/artwork pages and didn't find anything. Then I looked into the `react-zendesk` component (which is being used in `ZendeskWrapper` and it [includes](https://github.com/B3nnyL/react-zendesk/blob/master/src/index.js#L33) the script. So I am wondering why it needs to be added in the template?

Reply:
> I was looking to find where the zendesk script is loaded in the order/artwork pages and didn't find anything. Then I looked into the `react-zendesk` component (which is being used in `ZendeskWrapper` and it [includes](https://github.com/B3nnyL/react-zendesk/blob/master/src/index.js#L33) the script. So I am wondering why it needs to be added in the template?

Good question that I don't have an answer to other than to say that the library isn't mounting the script on our legacy pages so it needs to be injected in order to get the component to render.